### ISSUE TITLE
migrate/seed stub template conditionals

### ISSUE THREAD

**Issue by MadcapJake** [tgriesser/knex#911]
The CLI doesn't allow you to specifiy `tableName` so how would you utilize the conditonals in the stubs?

As in the `<% if (d.tableName) { %>` conditonal block that appears in the seed and migrate stubs.

**Comment by mercmobily** [comment]
You're absolutely right about this issue. Looking at the current codebase, the migration stub templates contain conditional logic like `<% if (d.tableName) { %>` but there's no way for users to actually provide a table name through the CLI.

The problem is in the migration stub template at `/lib/migrations/migrate/stub/js.stub`:

```javascript
exports.up = function(knex) {
  <% if (d.tableName) { %>
  return knex.schema.createTable("<%= d.tableName %>", function(t) {
    t.increments();
    t.timestamp();
  });
  <% } %>
};
```

But when you run `knex migrate:make create_users`, there's no `--table` option to populate that `tableName` variable, so the conditional is always false and you get an empty migration function.

**Here's how I would implement a fix:**

I'd add a `--table` option to both the `migrate:make` and `seed:make` commands. The changes would be:

1. **In `/bin/cli.js`**, add the table option to both commands:
```javascript
.command('migrate:make <name>')
.option('--table [name]', 'Specify the table name for the migration template')

.command('seed:make <name>') 
.option('--table [name]', 'Specify the table name for the seed template')
```

2. **Pass the table name to the migration generator:**
```javascript
if (opts.table) {
  configOverrides.variables = { tableName: opts.table };
}
```

3. **Update the seed stub templates** to also use the `tableName` variable consistently.

This would allow developers to run:
```bash
knex migrate:make create_users_table --table=users
```

And get a properly populated migration with the table name filled in.

**Alternative approach:**

If we don't want to add the CLI option, we should remove the conditional logic from the stub templates since it's currently dead code that confuses developers.

The current behaviour is definitely suboptimal - having template conditionals that can never be triggered creates a poor developer experience. This issue has merit and should be fixed.
